src/Okta.Sdk/IUser.cs

```diff
@@ -142,5 +144,19 @@ public partial interface IUser
         /// <param name="cancellationToken">The cancellation token</param>
         /// <returns>The <see cref="IFactor"/> response</returns>
         Task<IFactor> AddFactorAsync(AddTotpFactorOptions totpFactorOptions, CancellationToken cancellationToken = default(CancellationToken));
+
+        /// <summary>
+        /// Deactivates a user
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token</param>
+        /// <returns>The asynchronous task</returns>
+        [Obsolete("This method is deprecated, use DeactivateAsync(bool? sendEmail = false, CancellationToken cancellationToken = default(CancellationToken)) instead.", false)]
+        Task DeactivateAsync(CancellationToken cancellationToken = default(CancellationToken));
+
+        /// <summary>
+        /// Deactivates a user. Explicit overload to support backward compatibility.
```

Comment:
I like having comments to explain why these methods exist outside of the `.Generated.cs` files. 👍 Can you add "Explicit overload to support backward compatibility" to all of these methods you added?

Instead of placing it in the method description, add a `<remarks>` tag and put it there.